### PR TITLE
[msal-node-extensions]: Add default cache location

### PR DESCRIPTION
**Pull request by samuelkubai**
Add a default cache location in the cases where the user of the library does not provide a cache location.

### CODE REVIEW

File: change/@azure-msal-node-extensions-832ab65b-4ac3-41b1-a9d3-ad58efdc6892.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (jasonnutter) [inline change/@azure-msal-node-extensions-832ab65b-4ac3-41b1-a9d3-ad58efdc6892.json:2]: Given extensions is still in alpha, this should be `prerelease`.
```suggestion
  "type": "prerelease",
```

File: extensions/msal-node-extensions/src/utils/Constants.ts [hunk 2]
```diff
@@ -31,5 +31,32 @@ export const Constants = {
...
+    /**
+     * List of environment variables
+     */
+    ENVIRONMENT: {
+        HOME: "HOME",
+        LOGNAME: "LOGNAME",
+        USER: "USER",
+        LNAME: "LNAME",
+        USERNAME: "USERNAME",
+        PLATFORM: "platform",
```
Comment (sameerag) [inline extensions/msal-node-extensions/src/utils/Constants.ts:50]: everything else in caps and only `platform` is different. Any particular reason?

Comment (sameerag) [reply extensions/msal-node-extensions/src/utils/Constants.ts:50]: Nevermind. Is this how the system stores these in the `process.env()`?

**Review by sameerag — APPROVED** [review]
Lgtm

**Comment by samuelkubai** [comment]
Closing this pull request after discussions on the correct way forward.

**Pull request closed without merging**